linera-views/src/views/map_view.rs

```diff
@@ -838,11 +920,12 @@ where
         let mut hasher = sha3::Sha3_256::default();
         let mut count = 0;
```

Comment:
We should write `0u32` here. Otherwise the format is ambiguous

Reply:
Oh yes, and this impacts other containers, and it makes the change TestNet/DevNet breaking.